mm2src/coins/Cargo.toml

```diff
@@ -56,9 +56,9 @@ jsonrpc-core = "8.0.1"
 keys = { path = "../mm2_bitcoin/keys" }
 lazy_static = "1.4"
 libc = "0.2"
-lightning = "0.0.105"
+lightning = { git = "https://github.com/shamardy/rust-lightning", branch = "0.0.106" }
```

Comment:
Even newer rust lightning version still relies on old regex. Using personal forks is acceptable for WIP tasks unless the problems are resolved in the upstream.

Reply:
> 0.0.106 already exists in crates.io registry. :)

I will remove the dependency on the fork once this PR https://github.com/lightningdevkit/rust-lightning/pull/1509 is released :)

Reply:
@shamardy The https://github.com/lightningdevkit/rust-lightning/pull/1509 is merged. Should we switch to depending on the upstream repo? :)

Reply:
I prefer to wait for the v0.0.107 release since this is merged to the rust-lightning main branch that's similar to our dev branch and can break our code when other PRs get merged to it that need changes from our side.